src/shim/kmq/bo.cpp

```diff
@@ -25,6 +27,14 @@ flag_to_type(uint64_t bo_flags)
   return AMDXDNA_BO_INVALID;
 }
 
+void flush_cache_line(const char *cur) {
+  const int LINESIZE = 64;
```

Comment:
The x86_64 branch doesn't use this value. If it is ARM specific, move it to the ARM code.